app/views/shared/summary/expenses/index.html.haml

```diff
@@ -1,42 +1,33 @@
 %h3.govuk-heading-m
   = t(".header_#{vat ? 'with_vat' : 'without_vat'}")
 
-= govuk_table(class: current_user.case_worker? ? 'expenses-data-table' : nil, style: 'width:100%') do
-  = govuk_table_caption(class: 'govuk-visually-hidden') do
-    = t('.caption')
+= govuk_table(classes: current_user.case_worker? ? 'expenses-data-table app-table--responsive' : 'app-table--responsive', html_attributes: { style: 'width:100%' }) do |table|
```

Comment:
Similar to another comment. Is `style: 'width: 100%'` necessary here? I think it is included in the `govuk-table` class.